coalib/bearlib/abstractions/Linter.py

```diff
@@ -207,6 +207,9 @@ def check_prerequisites(cls):
             :return:
                 True if operational, otherwise a string containing more info.
             """
+            if not super(LinterBase, cls).check_prerequisites():
```

Comment:
Since coala is Python3-only, ` if not super().check_prerequisites(): ...` is enough here :)